### PR TITLE
Title for request

### PR DESCRIPTION
**Pull request by torin4fan**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,8 +8,11 @@
   <body>
     <!-- Так добавляются фрагменты в главные HTML-файлы страниц. -->
     <include src="./partials/example.html"></include>
-    <include src="./partials/problems.html"></include>
     <include src="./partials/footer.html"></include>
+    awdawd
```
Comment (torin4fan) [inline src/index.html:12]: what a fuck?

File: src/index.html [hunk 2]
```diff
@@ -8,8 +8,11 @@
...
+    awdawd
+    <h1>Title</h1>
+
```
Comment (torin4fan) [inline src/index.html:15]: test 2

**Comment by torin4fan** [comment]
1. You have some problems

File: src/index.html [hunk 3]
```diff
@@ -8,8 +8,11 @@
   <body>
     <!-- Так добавляются фрагменты в главные HTML-файлы страниц. -->
     <include src="./partials/example.html"></include>
-    <include src="./partials/problems.html"></include>
```
Comment (Maks1988-88-88) [inline src/index.html:11]: ок

**Review by Maks1988-88-88 — APPROVED** [review]

**Pull request merged**